backend/scripts/transfer-contract-ownership.ts

```diff
@@ -1,37 +1,117 @@
-import { SafeBulkWriter } from 'shared/safe-bulk-writer'
+import { createSupabaseDirectClient, pgp } from 'shared/supabase/init'
+
 import { runScript } from './run-script'
-import { convertUser } from 'common/supabase/users'
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+
+interface Args {
+  slugs: string[]
+  toUserId: string
+}
+
+interface User {
+  id: string
+  username: string
+  name: string
+  created_time: string
+}
+
+interface Contract {
+  id: string
+  data: any
```

Comment:
Using `any` isn't great code style. You should use follow the pattern of `getContracts` found elsewhere in the codebase.